html/capture-selion-events.html

```diff
@@ -0,0 +1,37 @@
+<div class="selion-content-div">
+  <h1 id="what-is-events" class="page-header">Working with SeLion events</h1>
```

Comment:
I think we need to be specific throughout this document and not confuse the user.. `events` and `element events` are not necessarily the same thing.